nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -223,7 +230,7 @@ define([
             if (this.tabWidget !== null) {
                 this.tabWidget.stop();
             }
-            this.tabWidget = new this.tabSet.tabs[tab].widget();
+            this.tabWidget = this.tabSet.tabs[tab].widget.make({bus: this.bus});
```

Comment:
Does every app widget require a bus?

Reply:
Yes, eventually. Well, possibly not, but this allows consistency in widget building.

Reply:
I mean, the info tab probably won't need one.